### ISSUE TITLE
Create a Sequential model in tensorflow

### ISSUE THREAD

**Issue by saumya0303** [tensorflow/tensorflow#37922]
Hi All,
         I have two models say M1, M2, I want to combine these models in such a way that the output of M1 is given as input to M2 and the gradient should be backpropagated till M1's input.

**Comment by khimraj** [comment]
@saumya0303, you can do this by following code.
```
model = tf.keras.Sequential()
model.add(M1)
model.add(M2)
M1,M2 = model.layers
M1.trainable = False
```

**Comment by nantha42** [comment]
> @saumya0303, you can do this by following code.
> 
> ```
> model = tf.keras.Sequential()
> model.add(M1)
> model.add(M2)
> M1,M2 = model.layers
> M1.trainable = False
> ```
What is reason for M1.trainable=False?

**Comment by khimraj** [comment]
@nantha42, My mistake as I misunderstood the query as i assumed backpropagation till M1's output instead of M1's input. I have corrected it.

**Comment by saumya0303** [comment]
is it necessary that both M1 and M2 should be present in Keras library??

**Comment by khimraj** [comment]
@saumya0303, It is not necessary. It will work with M1 and M2 models created by you also.

**Comment by saumya0303** [comment]
I am facing issue in adding a M1 model to a pre-trained DeepSpeech model as I am not able to backproagate till M1's input.

**Comment by khimraj** [comment]
@saumya0303, Can you share full code so that i can look at issue.

**Comment by saumya0303** [comment]
@khimraj **https://github.com/mozilla/DeepSpeech/blob/master/DeepSpeech.py** this links correspond to the deepspeech code before giving audio as input I want to add a layer.

**Comment by jvishnuvardhan** [comment]
This is not Build/Installation or Bug/Performance issue. Please post this kind of support questions at [Stackoverflow](https://stackoverflow.com/questions/tagged/tensorflow). There is a big community to support and learn from your questions. GitHub is mainly for addressing bugs in installation and performance. Thanks!

**Comment by saumya0303** [comment]
Thanks for the update.
